.circleci/cloudformation.yml

```diff
@@ -0,0 +1,108 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
I really ought to try playing with one of these on my own to wrap my head around it.

Would you suggest any particular CF tutorials/references?

Reply:
This documents every resource and property https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-template-resource-type-ref.html

I constantly have 3-4 of these tabs open at a time. Sometimes figuring stuff out concretely involves trial and error (e.g. do I provide the ARN or Name for this property?) Cloudformation is not perfect.

I think my first step would be to deploy an EC2 instance with Cloudformation. It requires you to define a lot of stuff, but it's a good kick-the-tires demo of CFTs

Reply:
https://github.com/awslabs/aws-cloudformation-templates I draw inspiration from these often